common/scala/build.gradle

```diff
@@ -24,6 +24,9 @@ plugins {
 }
 
 ext.dockerImageName = 'scala'
+if(System.getProperty("os.arch").toLowerCase(Locale.ENGLISH).startsWith("aarch")) {
```

Comment:
If the architecture of OS is m1 Mac, it uses the Dockerfile with the `.arm` suffix.